packages/core/src/App/Containers/AccountSignupModal/account-signup-modal.jsx

```diff
@@ -90,11 +109,18 @@ const AccountSignup = ({ enableApp, isModalVisible, clients_country, onSignup, r
     };
 
     const validateSignupPassthrough = values => validateSignup(values, residence_list);
+
     const onSignupPassthrough = values => {
         const index_of_selection = residence_list.findIndex(
             item => item.text.toLowerCase() === values.residence.toLowerCase()
         );
 
+        if (isEuResident === false) {
+            values.email_consent = true;
+        }
+
+        values.email_consent = values.email_consent ? 1 : 0;
```

Comment:
Can we just set it on line 119 directly? Or is there any other place where the value can become `true`?

Reply:
Unfortunately, I can only call the if statement here in this file (sofar I tried and know), in order to flip the email_consent boolean if I want to see if the user selected an EU country or not. However, I did determine the value type (line 122) now in the client-store file. Perhaps, that's better? If not, I can change it back again.